utils/constants.js

```diff
@@ -0,0 +1,9 @@
+const LOGS_ENDPOINT =
+  'https://us-central1-test-947g5.cloudfunctions.net/logs/log';
```

Comment:
Needs to be encrypted.

Reply:
Using a cryptography library or environment variables?